extensions/bitwarden/src/search.tsx

```diff
@@ -147,94 +174,95 @@ function ItemListItem(props: {
       subtitle={item.login?.username || undefined}
       actions={
         <ActionPanel>
-          {item.login?.password ? <PasswordActions password={item.login.password} /> : null}
-          {item.login?.username ? <UsernameAction username={item.login.username} /> : null}
-          {item.login?.totp ? (
-            <ActionPanel.Item
-              shortcut={{ modifiers: ["cmd"], key: "t" }}
-              title="Copy TOTP"
-              icon={Icon.Clipboard}
-              onAction={() => copyTotp(sessionToken, item.id)}
-            />
-          ) : null}
-          {item.notes ? (
-            <Action.Push
-              title="Show Secure Note"
-              icon={Icon.TextDocument}
-              target={
-                <Detail
-                  markdown={codeBlock(item.notes)}
-                  actions={
-                    <ActionPanel>
-                      <Action.CopyToClipboard title="Copy Secure Notes" content={item.notes} />
-                    </ActionPanel>
-                  }
-                />
-              }
-            />
-          ) : null}
-          <ActionPanel.Submenu
-            shortcut={{ modifiers: ["cmd", "shift"], key: "c" }}
-            icon={Icon.Clipboard}
-            title="Copy Property"
-          >
-            {Object.entries({
-              username: login?.username,
-              notes,
-              ...card,
-              ...identity,
-              ...fieldMap,
-              ...uriMap,
-            }).map(([title, content], index) =>
-              content ? (
-                <Action.CopyToClipboard key={index} title={titleCase(title)} content={content as string | number} />
-              ) : null
-            )}
-          </ActionPanel.Submenu>
-          <Action.Push
-            title={"Show Details"}
-            icon={Icon.Text}
-            shortcut={{ modifiers: ["cmd"], key: "i" }}
-            target={
-              <Detail
-                markdown={codeBlock(tree)}
-                actions={
-                  <ActionPanel>
-                    <Action.CopyToClipboard content={tree} />
-                  </ActionPanel>
+          <ActionPanel.Section>
+            {item.login?.password ? <PasswordActions password={item.login.password} /> : null}
+          </ActionPanel.Section>
+          <ActionPanel.Section>
+            {item.login?.username ? (
+              <Action.CopyToClipboard
+                title="Copy Username"
+                content={item.login.username}
+                icon={Icon.Person}
+                shortcut={{ modifiers: ["cmd"], key: "u" }}
+                onCopy={exit}
+              />
+            ) : null}
+            {item.login?.totp ? (
+              <Action
+                shortcut={{ modifiers: ["cmd"], key: "t" }}
+                title="Copy TOTP"
+                icon={Icon.Clipboard}
+                onAction={() => {
+                  copyTotp(sessionToken, item.id);
+                  exit();
```

Comment:
`copyTotp` contains code to `closeMainWindow`. Could remove it from there and keep the `exit()` here.

Reply:
Nevermind, the `exit()` stops `copyTotp` from running here.

Reply:
Fixed !